### PR TITLE
Release 1.0.0 to MASTER

### LINKED ISSUE #123: Update OpenWeatherMap credentials in travis.yml
* `OWM_KEY`

### LINKED ISSUE #125: Replace historic weather requests with current
The calls to historic weather become not free. In the test we can call current weather api (instead of historic). This changes only testing, not the prod code.

### PR DESCRIPTION
**Pull request by lukeindykiewicz**
There are two more commits (already in master), that's why changelog shows more commits than there are commits in this PR.
Replace historic weather requests with current (close #125)
Update OpenWeatherMap credentials in travis.yml (close #123)

That's due to change to gitflow workflow in the middle of this "release"

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,12 @@
+Version 1.0.0 (2020-08-27)
+--------------------------
+Reformat code (close #128)
+Update scalafmt to newest (close #127)
+Bump scala-lru-map to 0.5.0 (close #122)
+Bump Circe to 0.13.0 (close #120)
+Replace historic weather requests with current (close #125)
+Update OpenWeatherMap credentials in travis.yml (close #123)
```
Comment (benjben) [inline CHANGELOG:8]: CHANGELOG should not contain `close`. Given that you're going to update it, please take the opportunity to update the release date.

File: build.sbt [hunk 2]
```diff
@@ -17,7 +17,7 @@ lazy val root = project
   .settings(
     name := "scala-weather",
     organization := "com.snowplowanalytics",
-    version := "0.5.0",
+    version := "1.0.0",
```
Comment (benjben) [inline build.sbt:20]: This should happen in `Prepare for release` commit.

**Review by benjben — CHANGES_REQUESTED** [review]
Looks good, just CHANGELOG and `Prepare for release` to update.

https://github.com/snowplow/scala-weather/pull/130/commits/5a764c13aef63fcb4206ed3d42267f3dc1c0403a contains some reformatting, do you have scalafmt enabled globally for your sbt?

**Comment by lukeindykiewicz** [comment]
> 5a764c1 contains some reformatting, do you have scalafmt enabled globally for your sbt?
I have format on save in my editor. 
That's the only way to have code formatted, but it accidetally makes this kind of things. I'll try to fix that.

**Review by benjben — APPROVED** [review]

**Pull request merged**